packages/contracts/src/Inventory.sol

```diff
@@ -151,7 +170,7 @@ contract Inventory is Ownable {
 
     // Transfers a card of the sender to the inventory, mints a soulbound inventory card to the
     // sender in return.
-    function addCard(address player, uint256 cardID) external delegated(player) {
+    function addCard(address player, uint256 cardID) external delegated(player) notInGame(player) {
```

Comment:
Theoretically, this could be allowed, but it might be good to forbid it because there is really no reason to do this, and it might make frontend management a bit easier (no need to monitor these events while in-game).

But if we do this, shouldn't we also prevent `addDeck` by symmetry? I'm leaning yes for now.

Reply:
yes yes, you are correct.
no reason to limit this as it doesn't affect the game play